utils/fill_in_results.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+# Copyright 2020 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+
+WORKDIR="work"
+
+
+# Check if Insights OCP is installed and properly initialized.
+# Please look into README.md from ccx-rules-ocp repository how to setup
+# Python virtual environment in order to be able to run insights-run command.
+function check_insights_availability() {
+    insights-run --help > /dev/null 2>&1 || { echo >&2 -e "${RED}Insights OCP needs to be installed and initialized. Aborting.${NC}"; exit 1; }
+    echo -e "${GREEN}Insights OCP seems to be initialized properly${NC}"
+}
+
+
+# Decompress input file taken from Kraken or any other external source
+function decompress_input() {
+    echo -e "${BLUE}Decompressing input file $1${NC}"
+    tar xvfz "$1"
+    echo -e "${GREEN}Done${NC}"
+}
+
+
+# Input tarball has very strange directory structure (ask other teams why on Earth do they need it)
+# We just move all files from this structure into one "flat" workdir
+function move_input_to_workdir() {
+    find . -name "*.tar.gz" -exec mv -i -- {} "$1" \;
+}
+
+
+# Remove now empty temporary directories
+function cleanup_temp_directories() {
+    # shellcheck disable=SC2045,SC2035
+    for temp_directory in $(ls -1 -d */)
```

Comment:
hope nobody will add a space between `*` and `/` :)

Reply:
yup, and run it with sudo :)